samples/bluetooth/central_uart/src/main.c

```diff
@@ -361,11 +363,7 @@ static void connected(struct bt_conn *conn, uint8_t conn_err)
 			bt_conn_unref(default_conn);
 			default_conn = NULL;
 
-			err = bt_scan_start(BT_SCAN_TYPE_SCAN_ACTIVE);
-			if (err) {
-				LOG_ERR("Scanning failed to start (err %d)",
-					err);
-			}
+			k_work_reschedule(&scan_connect_work, CONNECT_TO_PEER_TIMEOUT);
```

Comment:
Consider `(void)` cast to indicate that it cannot fail

Reply:
btw. Why do we delay scan start here? Wouldn't a work without delay be good enough? (also in other places)